My-Portfolio Introduction branch

Marlyn-Mayienga opened the pull request:
# # My_Profile Introduction page
In this module we:
-created the html file
-created the style.css file
-modified the README file to describe the current project
-created the introduction page to the project in html
-styled the page to meet the specified CSS specifications

Marlyn-Mayienga committed:
created the my-introduction branch
created the html file
created the style.css file
modified the README file to describe the current project
created the introduction page to the project in html
styled the page

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,107 @@
+                <div class="technologies-title">Languages</div>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
```
- [ ] This content should be in `section tag` for best semantic HTML.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,107 @@
+    <main class="content">
+        <h1 class="name">Marlyn Mayienga</h1>
+        <img class="first-img" src="../My_Portfolio/circle 2 header.svg" alt="circle 2 header">
+        <h1>Software Developer</h1>
```
- [ ] Kindly wrap this `h1` tag with another `hx` tag like `h2`, `h3` etc because only for heading do we use the `h1` tag.

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,107 @@
+                <div class="technologies-item">
+                    <div class="technologies-title">Skills</div>
+                    <ul>
+                        <li>Codekit</li>
+                        <li>GitHub</li>
+                        <li>CodePEn</li>
+                        <li>GitLab</li>
+                        <li>Terminal</li>
+                    </ul>
```
- [ ] If you see the Figma design then you see that this content should be in the next line. so kindly fix it.
- [ ] Kindly make the `work` para just after the social icons and remove the one because there are two `work ` para in left and right corner
-  Your design
![his design](https://user-images.githubusercontent.com/81727988/164343845-5412dc4d-2ef1-45d6-b1ab-0fa5c6111e5e.png)
- Figma
![figma design](https://user-images.githubusercontent.com/81727988/164343867-141cbe87-90e4-486c-b401-db28169e3ef5.png)
a design

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,107 @@
+
+    <main class="content">
+        <h1 class="name">Marlyn Mayienga</h1>
+        <img class="first-img" src="../My_Portfolio/circle 2 header.svg" alt="circle 2 header">
```
- [ ] This image is not displaying in the project so kindly fix it.
- [ ] your `hr` tag is too big kindly fix it.
-
![img](https://user-images.githubusercontent.com/81727988/164344302-0fe4912b-0757-4e3c-941d-71461b7f2bda.png)

roshan-bajgain reviewed (changes requested):
Hi @Marlyn-Mayienga, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [ ]  You have a conflict on the pull request so the linter is not showing kindly resolve the conflict and make changes and use the same pull request for the next review.
 ![conflict](https://user-images.githubusercontent.com/81727988/164344620-70a7a499-6f0c-4c30-9ee6-b57f8badd981.png)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Marlyn-Mayienga committed:
fixed the linters to make sure all passed the tests

Marlyn-Mayienga committed:
made the following changes:
fixed the position of the images on the mobile view
corrected the main tag to section tag
corrected the h1 tag to h3
corrected the footer for the mobile view

Marlyn-Mayienga commented:
# # README Conflict error
Hello, I seem to be having an issue with the GitHub readme (that is causing the conflict)  file yet my vscode readme file has no errors. I followed the Microverse template given. 
 Any advice on why this is the case?
I tried attaching the image below if you are able to access it.

![image](https://user-images.githubusercontent.com/32592856/164541949-523916e4-22a5-4792-841f-6ec50b8dc6af.png)
![vscode](https://user-images.githubusercontent.com/32592856/164542194-da00a84f-65c6-4e27-ae99-2774313fa8e5.png)

Marlyn-Mayienga committed:
Merge branch 'master' into my-introduction

Marlyn-Mayienga committed:
corrected the footer and linter errors

Marlyn-Mayienga committed:
Merge branches 'my-introduction' and 'my-introduction' of https://github.com/Marlyn-Mayienga/My_Portfolio into my-introduction

Marlyn-Mayienga committed:
modified the html file

Marlyn-Mayienga committed:
fixed errors on the image

Marlyn-Mayienga committed:
fixed the h1 and h3 tags

Marlyn-Mayienga committed:
fixed the paragraph section padding

AmaduKamara reviewed (approved):
# Status: Approved 🟢🟢🟢 

Hi @Marlyn-Mayienga,

![ClappingLeonardoDicaprioGIF](https://user-images.githubusercontent.com/50941074/164555227-5af3be84-47ee-41c2-9b62-80bfa6a808fa.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
